libs/blocks/marketo/marketo.css

```diff
@@ -29,25 +32,37 @@
 }
 
 .marketo .mktoForm {
+  display: flex;
+  flex-flow: row wrap;
+  justify-content: space-between;
+  column-gap: 4.6%;
+  align-items: flex-start;
+  max-height: 400px;
+  overflow: hidden;
   font-size: 16px;
   font-weight: bold;
   width: 100%!important;
+  visibility: hidden;
+  opacity: 0;
+}
+
+.marketo.success .marketo-title,
+.marketo.success .marketo-description,
+.marketo.success form.mktoVisible.mktoForm {
+  display: none;
 }
 
-.marketo form.mktoForm--fade-in {
+.marketo form.mktoForm.mktoForm--fade-in {
+  transition: opacity 0.1s ease-in;
   visibility: hidden;
   opacity: 0;
 }
 
-.marketo form.mktoForm--fade-in.mktoVisible {
-  display: flex;
-  flex-flow: row wrap;
-  justify-content: space-between;
-  column-gap: 4.6%;
-  align-items: flex-start;
+.marketo form.mktoForm.mktoForm--fade-in.mktoVisible {
+  max-height: 10000px;
```

Comment:
thinking about it, we may want to store this and `400px` as variables for readability into why these numbers were chosen?

Reply:
Good call, we can set a more accurate number than `400px`, `291px` for 3 rows or `369px` for 4 rows.
```css
  --marketo-form-placeholder-height: calc(78px * 3 + 57px); /* 3 rows + submit */
```